File: BE/lookey/src/main/java/com/project/lookey/cart/dto/CartListResponse.java
package com.project.lookey.cart.dto;

import com.fasterxml.jackson.annotation.JsonProperty;
import java.util.List;

public record CartListResponse(
        List<Item> items
) {
    public record Item(
            @JsonProperty("cart_id") Integer cartId,
            @JsonProperty("product_id") Long productId,
            @JsonProperty("product_name") String productName
    ) {}
}

File: FE/ConvenienceSightApp/app/src/main/java/com/example/lookey/data/model/CartListResponse.kt
package com.example.lookey.data.model

data class CartListResponse(
    val result: CartListResult
)

data class CartListResult(
    val items: List<CartItem>
)

data class CartItem(
    val cart_id: Int,
    val product_id: Long,
    val product_name: String
)

